developer-docs/server-installation/understanding-jenkins-scripts-jobs-parameters-and-variables.md

```diff
@@ -50,7 +50,7 @@ The following environment variables are used for Jenkins setup.
 
 |Parameter| Description|
 |---------------|-----------|
-|github_release_tag | Specify a tag name here if you want to build from a tag. Example - release-2.7.0. This will look for a tag named release-2.7.0 in the repository URL configured in the Jenkins job and checkout the code from this tag. This should not be confused with **public_repo_branch**. The **public_repo_branch** is used only to checkout the Jenkinsfile which has all the build logic.|
+|github_release_tag | Specify a tag name here if you want to build from a tag. Example - release-2.8.0. This will look for a tag named release-2.8.0 in the repository URL configured in the Jenkins job and checkout the code from this tag. This should not be confused with **public_repo_branch**. The **public_repo_branch** is used only to checkout the Jenkinsfile which has all the build logic.|
```

Comment:
|github_release_tag | Specify a tag name from which you want the build, example - release-2.8.0.  It looks for a tag named release-2.8.0 in the repository URL configured in the Jenkins job and checks out the code from this tag. This tag is different from **public_repo_branch** which is used only to check out the Jenkinsfile which has all the build logic.|